Formula/llvm.rb

```diff
@@ -582,24 +589,19 @@ def caveats
 
     # Testing mlir
     (testpath/"test.mlir").write <<~EOS
-      func @bad_branch() {
-        br ^missing  // expected-error {{reference to an undefined block}}
-      }
-    EOS
-    system "#{bin}/mlir-opt", "--verify-diagnostics", "test.mlir"
+      func.func @main() {return}
 
-    (testpath/"scanbuildtest.cpp").write <<~EOS
-      #include <iostream>
-      int main() {
-        int *i = new int;
-        *i = 1;
-        delete i;
-        std::cout << *i << std::endl;
-        return 0;
-      }
+      // -----
+
+      // expected-note @+1 {{see existing symbol definition here}}
+      func.func @foo() { return }
+
+      // ----
+
+      // expected-error @+1 {{redefinition of symbol named 'foo'}}
+      func.func @foo() { return }
     EOS
-    assert_includes shell_output("#{bin}/scan-build clang++ scanbuildtest.cpp 2>&1"),
-      "warning: Use of memory after it is freed"
```

Comment:
I have no idea what's wrong with `scan-build`, so I've removed the failing test.

Someone can help fix it if they need `scan-build`.

Reply:
I can't reproduce this failure locally, so I hope it's gone away. I've put back the test for now.